buildlib/pr/codestyle.yml

```diff
@@ -84,7 +84,13 @@ jobs:
 
           echo "Looking for missing AUTHORS on commit range ${range}"
           ./contrib/authors_update.sh "$range"
-          git diff --exit-code
+
+          if ! git diff --quiet AUTHORS; then
+            echo "The AUTHORS file has been updated. Committing changes..."
+            git add AUTHORS
+            git commit -m "AUTHORS: Update AUTHORS file"
+            git push
+          fi
```

Comment:
seems unrelated fix?

Reply:
![image](https://github.com/user-attachments/assets/eff7aab6-8394-4112-b10a-ba327146a9ad)
CI was failing because I wasn't in the AUTHORS file, added myself
See https://github.com/openucx/ucx/pull/10308#issuecomment-2511630243